### PR TITLE
close #3.

### LINKED ISSUE #3: Merge ECP-SDK work from OESS-99.

### PR DESCRIPTION
**Pull request by hyoklee**

### CODE REVIEW

File: BITGROOM/CTestConfig.cmake [hunk 1]
```diff
@@ -0,0 +1,30 @@
+## This file should be placed in the root directory of your project.
+## Then modify the CMakeLists.txt file in the root directory of your
+## project to incorporate the testing dashboard.
+## # The following are required to uses Dart and the Cdash dashboard
+##   ENABLE_TESTING()
+##   INCLUDE(CTest)
+set (CTEST_PROJECT_NAME "HDF5PLUGINS")
+set (CTEST_NIGHTLY_START_TIME "18:00:00 CST")
+
+set (CTEST_DROP_METHOD "http")
```
Comment (byrnHDF) [inline BITGROOM/CTestConfig.cmake:10]: https

**Comment by byrnHDF** [comment]
I believe you used one of the other filters as a template, which one would be helpful in case I know of an issue. Because of the number of files involved Which ones you changed and how would be useful.

**Comment by byrnHDF** [comment]
Actually I see this is two filters, can we split this into two PRs - one for each filter?

File: BITGROOM/configure.ac [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+# library that enables BitGrooming quantization as an HDF5 filter.
+# This file is modified from a configure.ac from the hdf5_plugin project.
+
+# Allen Byrne, Ed Hartnett 1/14/19
+# Charlie Zender 9/10/20
+
+# Initialize autoconf.
+AC_PREREQ(2.59)
+AC_INIT(H5BGR, 1.0, nco-bugs@lists.sourceforge.net)
+AC_CONFIG_HEADER([config.h])
```
Comment (byrnHDF) [inline BITGROOM/configure.ac:13]: config.h will need a prefix, "bitgroom_config.h"

**Comment by hyoklee** [comment]
> I believe you used one of the other filters as a template, which one would be helpful in case I know of an issue. Because of the number of files involved Which ones you changed and how would be useful.

https://confluence.hdfgroup.org/display/OESS/Implementation

File: BITGROOM/configure.ac (hunk 2 above)
Comment (hyoklee) [reply BITGROOM/configure.ac:13]: I won't change unless it's essential for cmake build.

Comment (byrnHDF) [reply BITGROOM/configure.ac:13]: Yes, otherwise it will override other config.h files when building other files.

**Comment by byrnHDF** [comment]
> Actually I see this is two filters, can we split this into two PRs - one for each filter?

We will need to split this into two PRs because all the files are new and hard to review.

**Review by byrnHDF — COMMENTED** [review]
ZSTD has been implemented seperatetly.

**Comment by hyoklee** [comment]
> ZSTD has been implemented seperatetly.

What do you mean?

**Comment by hyoklee** [comment]
I close this PR based on the last @byrnHDF's comment.

**Pull request closed without merging**